cmd/zetatool/filterdeposit/btc.go

```diff
@@ -0,0 +1,159 @@
+package filterdeposit
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/zeta-chain/zetacore/cmd/zetatool/config"
+)
+
+var btcCmd = &cobra.Command{
+	Use:   "btc",
+	Short: "Filter inbound btc deposits",
+	Run:   FilterBTCTransactions,
```

Comment:
Instead of using `log.Fatal`
Can we use
```
RunE: FilterBTCTransactions
```
and return a error in the functions

Same comments for the other commands